### DISCUSSION TITLE
Idea for a patch to allow borderless operation

### DISCUSSION THREAD

**Discussion by skaface** [bakkeby/patches#53] [Ideas]
I have always wanted to run DWM without borders, I have a 4k monitor and would like to display multiple 1920x1080 windows at full resolution beside each other or in a 4x4 grid. (For example when showing remote desktops, it makes a big difference when the window size is reduced by the couple of pixels of border width)
I have modified the code to make the top bar float over the top of windows with the transparency patch applied so you can still see the windows contents below the bar.

I was thinking of showing window focus by putting a small square of maybe 4x4 pixels in the lower corner of the active window.

I don't think this would be too difficult to achieve but my C programming skills are lacking. I have a good understanding of how DWM functions by looking through the source and modifying pre-existing code, but writing my own modifications are unfortunately far beyond my skill level!

**Comment by bakkeby** [comment]
If you use a separate window to show the focus then you could use a simplified version of this example patch to achieve this. 
https://github.com/bakkeby/patches/blob/master/dwm/dwm-focusborder-6.3.diff

**Comment by bakkeby** [comment]
Here is an example patch that adds a small square in the bottom right corner of the window.
https://github.com/bakkeby/patches/blob/master/dwm/dwm-focuspixel-6.3.diff

**Reply by skaface** [reply]
thank you so much for your help, that works well. I have donated some beer/coffee money to your dwm-flexipatch project
